src/main/java/de/retest/web/RecheckSeleniumAdapter.java

```diff
@@ -86,7 +86,7 @@ public RootElement convertToPeers( final Map<String, Map<String, Object>> data,
 	@Override
 	public DefaultValueFinder getDefaultValueFinder() {
 		// TODO DefaultValueFinder is just a stub.
-		return ( identifyingAttributes, attributesKey ) -> null;
+		return ( identifyingAttributes, s, serializable ) -> false;
```

Comment:
Can't we return `DefaultValuesProvider` now?

Reply:
This should be a separat commit. This is only about resolving compile errors. Small steps.

Reply:
Indeed 👍